Source/JavaScriptCore/inspector/protocol/DOM.json

```diff
@@ -223,7 +223,8 @@
                 { "name": "codec", "type": "string", "description": "The codec string of the primary audio track. (E.g., \"hvc1.1.6.L123.B0\")" },
                 { "name": "humanReadableCodecString", "type": "string", "description": "A human readable version of the `codec` parameter."},
                 { "name": "numberOfChannels", "type": "integer", "description": "The number of audio channels in the primary audio track." },
-                { "name": "sampleRate", "type": "number", "description": "The sample rate of the primary audio track in hertz." }
+                { "name": "sampleRate", "type": "number", "description": "The sample rate of the primary audio track in hertz." },
+                { "name": "isProtected", "type": "boolean", "description": "Whether the track contains protected contents" }
```

Comment:
id recommend making this optional since it seems like it not being provided is indistinguishable from `false` (i.e. avoid sending unnecessary data)